tests/graphics/bitmap.cpp

```diff
@@ -57,6 +58,23 @@ typedef wxNativePixelData wxNative32PixelData;
 // tests
 // ----------------------------------------------------------------------------
 
+TEST_CASE("BitmapTestCase::Monochrome", "[bitmap][monochrome]")
+{
+    wxBitmap color;
+    color.LoadFile("horse.bmp", wxBITMAP_TYPE_BMP);
+    REQUIRE(color.IsOk());
+    REQUIRE(color.GetDepth() == 32);
+
+    wxImage imgQuant = color.ConvertToImage();
+    wxBitmap bmpQuant(imgQuant, 1);
+    bmpQuant.SaveFile("quantize.bmp", wxBITMAP_TYPE_BMP);
```

Comment:
Please use `TestFile` for this file, so it's not left lying around.

It would also be more clear to call it `mono_horse.bmp`, I think.